at_client/lib/src/preference/at_client_config.dart

```diff
@@ -0,0 +1,13 @@
+/// The class contains all the client configurations.
+class AtClientConfig {
+  static final AtClientConfig _singleton = AtClientConfig._internal();
+
+  AtClientConfig._internal();
+
+  factory AtClientConfig.getInstance() {
+    return _singleton;
+  }
+
+  /// Represents the at_client version.
+  final String atClientVersion = '3.0.36';
```

Comment:
Maintenance of this needs to be automated - maybe through using build_runner. At a minimum, you need to add a VERY VISIBLE comment to pubpsec.yaml stating that when the package version is increasing, then this constant needs to be changed accordingly.

Reply:
@gkc: For now, to get going, will add a comment in pubspec.yaml. And keep exploring the alternatives to automate the capturing the at_client version. Does this sound's good?

Reply:
👍 thanks Sita